src/rpc.ts

```diff
@@ -76,7 +74,7 @@ export class WatchClient {
    * last compaction revision.
    */
   public watch(): Promise<IDuplexStream<IWatchRequest, IWatchResponse>> {
-    return this.client.getConnection('Watch').then(cnx => cnx.watch());
+    return this.client.getConnection('Watch').then(cnx => (<any> cnx.client).watch());
```

Comment:
Why the any cast? Missing typings? Tried adding patching them in via interface augmentation?